microbit/v2/source/main.cpp

```diff
@@ -46,6 +47,19 @@ void sendString(ManagedString s)
     uart->send(s);
 }
 
+/**
+ * @brief Sends an array of bytes with UART
+ * 
+ * @param s The bytes
+ */
+void sendBytes(uint8_t s)
```

Comment:
This sends a single byte, not an array of bytes. Should this be a uint8_t* and a length? Then we can also send the size we which will need.